test/e2e/fixture-builder.js

```diff
@@ -289,7 +289,6 @@ function defaultFixture() {
         allTokens: {},
         detectedTokens: [],
         ignoredTokens: [],
-        suggestedAssets: [],
```

Comment:
I think I mentioned on the core PR also, but do we have a policy to remove old unused user data using migrations, or leave it to linger?

Reply:
I'm going to ask in one of the dev channels.

Reply:
Turns out a migration to get rid of the state in existing installations would be desirable in this case.